git/git_test.go

```diff
@@ -0,0 +1,129 @@
+package git_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/paketo-buildpacks/samples/tests"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+var builders tests.BuilderFlags
+var suite spec.Suite
```

Comment:
```suggestion
```

Do we need this? Looks like it shadows the `suite` variable that gets defined in the `TestNginx` method.